tests/components/amcrest/test_config_flow.py

```diff
@@ -0,0 +1,268 @@
+"""Test the Amcrest config flow."""
+from unittest.mock import patch
+
+from amcrest import AmcrestError, LoginError
+
+from homeassistant import config_entries
+from homeassistant.components.amcrest.const import DOMAIN
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+
+from tests.common import MockConfigEntry
+
+
+async def test_form(hass: HomeAssistant) -> None:
```

Comment:
You should create a `mock_setup_entry` fixture in a local `conftest.py` file, and auto-use it here with `pytestmark`

```suggestion
pytestmark = pytest.mark.usefixtures("mock_setup_entry")


async def test_form(hass: HomeAssistant, mock_setup_entry: AsyncMock) -> None:
```